workers/unity/Packages/com.improbable.gdk.transformsynchronization/Systems/PositionSendSystem.cs

```diff
@@ -62,7 +62,7 @@ protected override void OnUpdate()
                 var entityId = positionData.SpatialEntityIds[i].EntityId;
 
                 var update = new SchemaComponentUpdate(position.ComponentId);
-                Position.Serialization.Serialize(position, update.GetFields());
+                Position.Serialization.SerializeUpdate(position, update);
```

Comment:
We should really make helper functions for this. There is no reason a user has to touch `Serialization`.

Reply:
Not for this PR though.

Reply:
Sort of like a:

`ComponentUpdate CreateComponentUpdate(Position.Component component)` sort of thing?

Reply:
Yes. Could take it a step further and make a wrapper in the components send system that also sends it in the connection. Ideally users never need to touch that in these classes too. protected methods > protected fields.